include/PassType.h

```diff
@@ -0,0 +1,61 @@
+#ifndef PASS_H
+#define PASS_H
+
+#include "RunAnalysis.h"
+#include "Utils.h"
+#include "CFG.h"
+
+#include <set>
+#include <fstream>
+
+// mapping between LLVM intermediate variables (e.g., %6, %7, %8) and variables visible in the code (e.g., %str). Used to find all local must-aliases, though it is not expected to find all of them
+typedef std::map<std::string, std::string> AliasMap;
+
+struct MaybeUninitMethodsSet {
+    std::set<std::string> methodsSet; 
+    bool setInitialized; 
+};
+
+// mapping for an entire program; first string is branch name, second string is var name 
```

Comment:
Why are branch names used as keys? This might be because I don't fully understand what LLVM is giving you, but it goes against my expectations that you'd use branch names at all to identify an estimate: I'd expect that there'd be an entry in this map for every program point, instead.

Reply:
I think here we can say branch names are program points; each branch name holds the must call/called methods estimate for each variable after all the code within that branch has run.